Cargo.toml

```diff
@@ -20,7 +20,7 @@ prost = "0.11"
 prost-types = "0.11"
 serde = "1.0"
 snafu = "0.7"
-stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", tag = "0.46.0" }
+stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", tag = "0.48.0" }
```

Comment:
`stackable-operator` should be upgraded to the latest version.

Reply:
64520ae18639d9fdf744e437225301f53fbf6282